Add acronym to first occurrence (#20356)
dotnet/docs: docs/standard/net-standard.md

.NET Standard is not specific to any one .NET implementation, nor does it match the versioning scheme of any of those runtimes.

> APIs added to any of the implementations (such as, .NET Framework, .NET Core and Mono) can be considered as candidates to add to the specification, particularly if they are thought to be fundamental in nature. New [versions of .NET Standard](https://github.com/dotnet/standard/blob/master/docs/versions.md) are created based on .NET implementation releases, enabling you to target new APIs from a .NET Standard PCL (Portable Class Libraries). The versioning mechanics are described in more detail in [.NET Core Versioning](../core/versions/index.md).

Reviewer: ```suggestion
APIs added to any .NET implementation (for example, .NET Framework, .NET Core, and Mono) can be considered as candidates to add to the specification, particularly if they are thought to be fundamental in nature. New [versions of .NET Standard](https://github.com/dotnet/standard/blob/master/docs/versions.md) are created based on .NET implementation releases, enabling you to target new APIs from a .NET Standard portable class library (PCL). The versioning mechanics are described in more detail in [.NET Core Versioning](../core/versions/index.md).
```

Reviewer: I changed the first sentence and switched the acronym with the definition.
